rdkit/Chem/Fraggle/FraggleSim.py

```diff
@@ -197,8 +171,39 @@ def select_fragments(f_smi, ftype, hac):
     raise NotImplementedError('Invalid fragmentation type {0}'.format(ftype))
 
 
+def isValidRingCut(mol):
+  """ to check is a fragment is a valid ring cut, it needs to match the
+  SMARTS: [$([#0][r].[r][#0]),$([#0][r][#0])]
+
+  For valid ring cuts, the function returns the number of atoms. Otherwise 0. """
+  # At this point, the molecule requires the identification of rings, so we need to sanitize
+  Chem.SanitizeMol(mol, Chem.SanitizeFlags.SANITIZE_SYMMRINGS)
```

Comment:
If the only sanitize operation you need is the ring finding, you could just call the ring finding code directly.